ios/fastlane/Fastfile

```diff
@@ -16,8 +16,60 @@
 default_platform(:ios)
 
 platform :ios do
-  desc "Description of what the lane does"
-  lane :custom_lane do
-    # add actions here: https://docs.fastlane.tools/actions
+
+  private_lane :apple_auth do |options|
+    ensure_env_vars(
+        env_vars: ["APP_STORE_CONNECT_API_KEY_ID", "APP_STORE_CONNECT_API_ISSUER_ID", "APP_STORE_CONNECT_API_KEY_CONTENT"]
+    )
+
+    app_store_connect_api_key(
+        key_id: ENV['APP_STORE_CONNECT_API_KEY_ID'],
+        issuer_id: ENV['APP_STORE_CONNECT_API_ISSUER_ID'],
+        key_content: Base64.decode64(ENV['APP_STORE_CONNECT_API_KEY_CONTENT'])
+    )
   end
+
+  desc "Create a release build"
+  lane :build do |options|
+    setup_circle_ci
```

Comment:
What does this do?

Reply:
-Creates a new temporary keychain for use with match
-Switches match to readonly mode to not create new profiles/cert on CI
-Sets up log and test result paths to be easily collectible